modules/map/hdmap/adapter/xml_parser/objects_xml_parser.cc

```diff
@@ -238,10 +239,126 @@ Status ObjectsXmlParser::ParsePNCJunctions(
         return Status(apollo::common::ErrorCode::HDMAP_DATA_ERROR, err_msg);
       }
       RETURN_IF_ERROR(UtilXmlParser::ParseOutline(*outline_node, polygon));
+
+      RETURN_IF_ERROR(ParsePassageGroup(*sub_node, &pnc_junction));
+
       pnc_junctions->emplace_back(pnc_junction);
     }
+
     sub_node = sub_node->NextSiblingElement("object");
   }
+
+  return Status::OK();
+}
+
+Status ObjectsXmlParser::ParsePassageGroup(const tinyxml2::XMLElement& xml_node,
+                                        PbPNCJunction* pnc_junction) {
+  CHECK_NOTNULL(pnc_junction);
+
+  auto sub_node = xml_node.FirstChildElement("passageGroup");
+  while (sub_node) {
+    std::string object_id;
+    std::string object_type;
+    PbPassageGroup *passage_group = pnc_junction->add_passage_group();
+    int checker =
+      UtilXmlParser::QueryStringAttribute(*sub_node, "id", &object_id);
+    if (checker != tinyxml2::XML_SUCCESS) {
+      std::string err_msg = "Error parse object type.";
+      return Status(apollo::common::ErrorCode::HDMAP_DATA_ERROR, err_msg);
+    }
+    passage_group->mutable_id()->set_id(object_id);
+
+    RETURN_IF_ERROR(ParsePassage(*sub_node, passage_group));
+
+    sub_node  = sub_node->NextSiblingElement("passageGroup");
+  }
+
+  return Status::OK();
+}
+
+Status ObjectsXmlParser::ParsePassage(const tinyxml2::XMLElement& xml_node,
+                                  PbPassageGroup* passage_group) {
```

Comment:
align your parameters across multiple lines.

Reply:
thanks for your review，I have fixed this issue.